src/main.js

```diff
@@ -114,56 +115,91 @@ class Game {
   }
 
   imageOnClick(idClicked) {
-    if (idClicked === this.prevClicked) return;
-    if (this.openedCards === 2) return;
-    this.cards[idClicked].open();
-    if (this.openedCards === 1) {
-      this.prevClicked = idClicked;
-      return;
+    if (this.fieldIsBlocked) return;
+    switch (this.openedCardsNum) {
+      case 0:
+        this.cards[idClicked].open();
+        return;
+      case 1:
+        if (this.isCardOpened(idClicked)) return;
+        this.cards[idClicked].open();
+        this.fieldIsBlocked = true;
+        this.checkCards();
+        break;
+      case 2:
+        break;
+      default:
     }
-    if (!this.cardsEqual(this.prevClicked, idClicked)) {
-      setTimeout(() => this.returnBackImage(idClicked), 1000);
+  }
+
+  isCardOpened(cardId) {
+    return parseInt(this.openedCards[0].div.id.slice(3), 10) === cardId;
```

Comment:
So, let's check what this line is doing:
- takes array of opened cards
- gets first item
- finds `div` on it
- finds `id` on div
- skips first 3 chars
- convert to int
- compares with given `cardId`

I think it's too much for one line